modules/pel/peltool/peltool.py

```diff
@@ -634,14 +636,18 @@ def extractAllPELsData(path: str, config: Config):
                 _, json_string = parsePEL(stream, config, False)
                 if json_string:
                     if not config.hex:
-                        allPELsData = allPELsData + json_string + ",\n"
+                        if firstPELPrinted:
+                            print(",")
+                        print(json_string, end = "", flush=True)
```

Comment:
Why are you flushing? I think you mentioned that `flush` does not make any improvement while printing into terminal, right?